.github/workflows/configure_osc.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+
+# This helper script creates the "osc" configuration file with OBS credentials
+
+CONFIG_FILE="$HOME/.config/osc/oscrc"
+
+# do not overwrite the existing config accidentally
+if [ -e "$CONFIG_FILE" ]; then
+    echo "ERROR: $CONFIG_FILE already exists"
+    exit 1
+fi
+
+TEMPLATE=$(dirname "${BASH_SOURCE[0]}")/oscrc.template
+mkdir -p $(dirname "$CONFIG_FILE")
+sed -e "s/@OBS_USER@/$OBS_USER/g" -e "s/@OBS_PASSWORD@/$OBS_PASSWORD/g" "$TEMPLATE" > "$CONFIG_FILE"
```

Comment:
```suggestion
mkdir -p $(dirname "$CONFIG_FILE")
cat > $CONFIG_FILE << EOT
[general]
apiurl = https://api.opensuse.org

[https://api.opensuse.org]
user=$OBS_USER
pass=$OBS_PASSWORD
credentials_mgr_class=osc.credentials.PlaintextConfigFileCredentialsManager
EOT
```
Not sure but for me it looks simpler...

Reply:
Let's leave this file as is, because I took it from upstream

Reply:
ok